packages/safety-suite-api/src/api/models/types.ts

```diff
@@ -115,23 +115,281 @@ export type OrderBy = {
 };
 
 export type ModelListOptions = {
+  /**
+   * Maximum number of models to return.
+   */
   limit?: number;
+  /**
+   * Options for sorting the returned models.
+   */
   orderBy?: OrderBy;
+  /**
+   * Index in the array of returned models.
+   */
   start?: number;
 };
 
 export type ModelOutputOptions = {
+  /**
+   * Will include `profilePictureUrl` in returned models if present.
+   */
   profilePicture?: boolean;
+  /**
+   * Will include confidential fields in returned models if permitted.
+   */
   protectedFields?: boolean;
+  /**
+   * Will include related models in returned models.
+   * `singleModel` will return only models with a singleModel relation.
```

Comment:
There are also `multiModel` and `children` options for `relationModels` parameter